app/controllers/concerns/idv/choose_id_type_concern.rb

```diff
@@ -61,9 +61,18 @@ def locals_attrs(presenter:, form_submit_url: nil)
       {
         presenter:,
         form_submit_url:,
-        disable_passports:,
+        disable_passports: disable_passports?,
         auto_check_value: disable_passports ? :drivers_license : selected_id_type,
       }
     end
```

Comment:
Why not use `disable_passports?` here? You could update the variable in this method to be set to that value.

```suggestion
disable_passports = disable_passports?
{
presenter:,
form_submit_url:,
disable_passports: disable_passports,
auto_check_value: disable_passports ? :drivers_license : selected_id_type,
}
```